pages/price-feeds/use-real-time-data/starknet.mdx

```diff
@@ -0,0 +1,176 @@
+---
+description: Consume Pyth Network prices in Starknet applications
+---
+
+import { Callout, Tabs } from "nextra/components";
+
+# How to Use Real-Time Data in Starknet Contracts
+
+This guide explains how to use real-time Pyth data in Starknet contracts.
+
+## Install the Pyth SDK
+
+Use the following dependency in your `Scarb.toml` file to use the latest Pyth Starkenet package:
+
+```toml copy
+[dependencies]
+pyth = { git = "https://github.com/pyth-network/pyth-crosschain.git", tag = "pyth-starknet-contract-v0.1.0"}
+```
+
+Pyth also provides a javascript SDK to interact with the Pyth contract on Starknet. You can install it using the following command:
+
+<Tabs items={["npm", "yarn"]}>
+  <Tabs.Tab>
+    ```sh copy npm install --save @pythnetwork/pyth-starknet-js ```
+  </Tabs.Tab>
+  <Tabs.Tab>```sh copy yarn add @pythnetwork/pyth-starknet-js ```</Tabs.Tab>
+</Tabs>
+
+## Write Contract Code
+
+The code snippet below provides an example module fetching the ETH/USD price from Pyth price feeds:
+
+```rust {2,17,47,55,64,71-73} copy
+use starknet::ContractAddress;
+use pyth::ByteBuffer;
+
+#[starknet::interface]
+pub trait IExampleMethod<T> {
```

Comment:
I would prefer to use `IExampleContract` instead of `IExampleMethod` (and the same applies to `example_method` and `ExampleMethod` below). In general, the trait represents all public methods of a contract, not a single method.